lib/bridge.js

```diff
@@ -1,12 +1,15 @@
 const { ethers } = require("hardhat");
 
-async function deployBridge(regenBridge, toucanRegistry) {
+async function deployBridge(tco2Factory, regenBridge, toucanRegistry) {
 	const Bridge = await ethers.getContractFactory("ToucanRegenBridge");
 	const bridge = await Bridge.deploy(regenBridge, toucanRegistry);
 
 	await bridge.deployed();
 	console.log("Bridge deployed to %s", bridge.address);
 
+	console.log(`Adding bridge contract address ${bridge.address} to allow list...`);
+	await tco2Factory.addToAllowlist(bridge.address);
```

Comment:
The reason I didn't add this here but instead in [`beforeEach`](https://github.com/regen-network/toucan-bridge/blob/b41af95ad0c1d93e561b2d907dae56f6961694dd/test/ToucanRegenBridge.js#L31) in the tests is to keep this script usable also for production use. As it stands, we cannot execute `addToAllowlist` like this in production because the tco2 factory is controlled by a multisig.

Reply:
I'd moving this out of the library and into the scripts that need to use it.

Reply:
OK I've moved that code to the deploy script, and renamed it to `deploy-local.js` to make it a bit more obvious that it's a deploy script for local use.